Examples/Swift/AppDelegate+CarPlay.swift

```diff
@@ -47,6 +47,17 @@ extension AppDelegate: CarPlayManagerDelegate {
         NavigationViewController.carPlayManagerDidEndNavigation(carPlayManager, window: window)
     }
     
+    func carPlayManagerFavoritesList(_ carPlayManager: CarPlayManager) -> CPListTemplate {
+        let mapboxSFItem = CPListItem(text: FavoritesList.POI.mapboxSF.rawValue,
```

Comment:
This could _totally_ wait until after MVP, but it might be nice to have the client-developer return a tuple `(name: String, location: CLLocationCoordinate2D)` instead of burdening the dev with creating the `CarPlay` model-objects themselves. 

Just a thought.

Reply:
👍

Reply:
That was very much the original idea. This is meant to be example code, but a more obvious factoring would help developers here.